VueJS/index.js

```diff
@@ -1,13 +1,72 @@
-import List from './components/List.js'
-import ListItem from './components/ListItem.js'
+// import List from './components/List.js'
+// import ListItem from './components/ListItem.js'
 
-new Vue({
-  el: '#app',
-  data: {
-    todos: [
-      { id: '1', message: 'Foo', },
-      { id: '2', message: 'Bar', },
-      { id: '3', message: 'Baz', }
-    ]
+Vue.component('list-item', {
+  props: ['todo'],
+  template: `
+      <div v-if="edit">
+        <li>
```

Comment:
```suggestion
        <li v-if="edit">
```
Put the if statement directly on the `<li` item. This is way more semantic.